examples/Exploring-different-models.ipynb

```diff
@@ -0,0 +1,805 @@
+    "\n",
+    "We define different deep learning-based model architectures, and then train and evaluate the models only in couple lines of code.\n",
+    "\n",
+    "### Learning objectives\n",
+    "- Preparing the data with NVTabular\n",
+    "- Training different deep learning-based recommender models with Merlin Models"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0f354e2a",
+   "metadata": {},
+   "source": [
+    "## Feature Engineering with NVTabular"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9eadd76d",
+   "metadata": {},
+   "source": [
+    "When we work on a new recommender systems, we explore the dataset, first."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "64c4b1d6",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2022-03-16 15:32:17.079946: I tensorflow/core/common_runtime/gpu/gpu_process_state.cc:214] Using CUDA malloc Async allocator for GPU: 0\n",
+      "2022-03-16 15:32:17.080101: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1525] Created device /job:localhost/replica:0/task:0/device:GPU:0 with 16254 MB memory:  -> device: 0, name: Quadro GV100, pci bus id: 0000:15:00.0, compute capability: 7.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "import os\n",
+    "os.environ[\"TF_GPU_ALLOCATOR\"]=\"cuda_malloc_async\"\n",
+    "import cudf\n",
+    "import glob\n",
+    "import gc\n",
+    "\n",
+    "import nvtabular as nvt\n",
+    "from nvtabular.ops import *\n",
+    "from example_utils import workflow_fit_transform\n",
+    "from example_utils import save_results\n",
+    "\n",
+    "from merlin.schema.tags import Tags\n",
+    "from merlin.schema import Schema\n",
+    "\n",
+    "import merlin.models.tf as mm\n",
+    "import merlin.models.tf.dataset as tf_dataloader\n",
+    "\n",
+    "from merlin.io.dataset import Dataset\n",
+    "from merlin.schema.io.tensorflow_metadata import TensorflowMetadata\n",
+    "from merlin.models.tf.blocks.core.aggregation import CosineSimilarity\n",
+    "\n",
+    "import tensorflow as tf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2f39ad9e",
+   "metadata": {},
+   "source": [
+    "First, we define our input and output paths."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "a6e0eaaa",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DATA_FOLDER = \"/workspace/data/\"\n",
+    "train_path = os.path.join(DATA_FOLDER, 'train/*.parquet')\n",
+    "test_path = os.path.join(DATA_FOLDER, 'test/*.parquet')\n",
+    "output_path = '/workspace/data/processed'"
```

Comment:
Should we use DATA_FOLDER as part of that path?

Reply:
Let's keep it as it is.